setup.py

```diff
@@ -284,7 +291,7 @@ def setup_package():
     else:
         if sys.version_info < (3, 6):
```

Comment:
```suggestion
        if sys.version_info < (3, 8):
```

Reply:
We might need someting similar to 

https://github.com/scikit-learn/scikit-learn/pull/22617/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R235-R239

for the conda-forge pypy build.

Reply:
Addressed concern in [`752700f` (#22617)](https://github.com/scikit-learn/scikit-learn/pull/22617/commits/752700f587ec689cb97f6b5960e2b5d93277ba21)